firmware/ci.c

```diff
@@ -669,7 +673,7 @@ static void video_matrix_list(void)
 
 static void video_matrix_connect(int source, int sink)
 {
-	if(source >= 0 && source <= VIDEO_IN_PATTERN)
+	if(source >= 0 && source <= VIDEO_IN_PCIE)
```

Comment:
We should add a `VIDEO_IN_MAX` or similar constant.

Reply:
done